### PR TITLE
Documentation parser/reader

### PR DESCRIPTION
**Pull request by Herteby**
I understand that the new documentation format is easier to maintain, but I found it a bit annoying to look through. Therefore I made this quick parser that gets all the sections and puts them together into one page!

I think if you enable github pages for this repo it should just work!

You can see it here: https://herteby.github.io/grapher/

### CODE REVIEW

**Comment by Herteby** [comment]
Another alternative could be the GitHub wiki feature, or readthedocs.org
Something that you don't manually have to compile is definitely nice.

**Comment by theodorDiaconu** [comment]
@Herteby just when I thought you couldn't impress me anymore, denormalize, graphical grapher, now this.

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+	<head>
+		<title>Grapher Documentation</title>
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
+		<script type="text/javascript" src="https://vuejs.org/js/vue.js"></script>
+		<script type="text/javascript" src="https://unpkg.com/marked@0.3.6"></script>
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+		<script type='text/javascript'>
+			window.onload = function(){
+				marked.setOptions({
```
Comment (theodorDiaconu) [inline docs/index.html]: Can you please properly indent the code ? Use prettier or something. And spaces instead of tabs

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
+<html>
+	<head>
+		<title>Grapher Documentation</title>
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
+		<script type="text/javascript" src="https://vuejs.org/js/vue.js"></script>
```
Comment (theodorDiaconu) [inline docs/index.html]: maybe use a specific version of vue ? So we don't head into other problems when this is updated in the future ?

**Comment by theodorDiaconu** [comment]
https://cult-of-coders.github.io/grapher/ any idea why it looks like this ?

**Comment by Herteby** [comment]
What do you mean? Looks fine to me

**Comment by theodorDiaconu** [comment]
It was a caching issue on my end. Cheers, looks great.

**Pull request merged**